TODO.md

```diff
@@ -436,3 +484,68 @@
   - http://kapadia.github.io/emscripten/2013/09/13/emscripten-pointers-and-pointers.html
   - this explains how to pass a pointer from a typearray to c++ 
   - this would avoid the dataHeap.set() - 43%
+
+---
+
+# *The following is a part of [TODO2.md](https://github.com/jeromeetienne/AR.js/blob/master/TODO2.md)*
```

Comment:
Change this link to https://github.com/jeromeetienne/AR.js/blob/8de374e725e4d9d9043aec137438fafdbad58e65/TODO2.md, so it will also work after the file has actually be deleted on `master` as well.

Reply:
Made a new commit. You can see the changes there.
Thanks for the feedback.